### PR TITLE
Extractor script part2 [BA-6380]

### PR DESCRIPTION
**Pull request by cjllanwarne**
Note to reviewers:
* The majority of the content of this PR is two metadata test cases. Feel free to skip over those.

Reminder to myself: 

- [x] Rebase onto develop when.`cjl_6346_extractor` merged.

Question for pythonistas:
* ~Is there a more canonical way to lay out python test cases~

A thought for later:
* Can we add python tests to our travis CI?

### CODE REVIEW

**Comment by cjllanwarne** [comment]
A thought probably for myself, probably less "later": can we stop running the full CI suite if the only changes are in `scripts` (like we do with docs)

File: scripts/metadata_comparison/test/resources/forkjoin_metadata.json [hunk 1]
```diff
@@ -0,0 +1,765 @@
...
+    },
+    {
+      "cromwellId": "cromid-1444cfc",
+      "cromwellVersion": "cromwell-version.conf-to-be-generated-by-sbt",
+      "description": "PickedUp",
+      "timestamp": "2020-04-16T21:22:53.327Z"
+    }
+  ],
+  "workflowRoot": "gs://cloud-cromwell-dev-self-cleaning/cromwell_execution/ci/forkjoin/515cfe12-1c50-4b93-ae77-b6675e87c0b8/"
+}
```
Comment (grsterin) [inline scripts/metadata_comparison/test/resources/forkjoin_metadata.json:765]: new line?

File: scripts/metadata_comparison/test/resources/subworkflow_hello_world_metadata.json [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+    },
+    {
+      "cromwellId": "cromid-e4f268e",
+      "cromwellVersion": "cromwell-version.conf-to-be-generated-by-sbt",
+      "description": "Finished",
+      "timestamp": "2020-04-16T20:51:22.878Z"
+    }
+  ],
+  "workflowRoot": "gs://cloud-cromwell-dev-self-cleaning/cromwell_execution/ci/main_workflow/f6cf8176-f7f9-458c-9bf7-e2b2a416e28e/"
+}
```
Comment (grsterin) [inline scripts/metadata_comparison/test/resources/subworkflow_hello_world_metadata.json:286]: new line

**Review by koenbnl — COMMENTED** [review]
<removed>

**Pull request merged**